mindjunkies/forums/tests/test_views.py

```
@@ -1,197 +1,186 @@
-        setattr(request, '_messages', messages)
-        
+        setattr(request, "_messages", messages)
+
         view = TopicSubmissionView.as_view()
-        response = view(
-            request, 
-            course_slug=self.course.slug,
-            module_id=self.module.id
-        )
-        
+        response = view(request, course_slug=self.course.slug, module_id=self.module.id)
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)